lib/xwax/pitch_kalman.h

```diff
@@ -58,23 +59,24 @@ struct pitch_kalman {
 
     /* Tresholds of the innovation quantity for the mode switches */
```

Comment:
hmmm still not quite sure what is meant by innovation again

```suggestion
    /* Thresholds of the innovation quantity for the mode switches */
```